packages/vuetify/src/components/VColorPicker/util/index.ts

```diff
@@ -144,9 +144,10 @@ export function extractColor (color: VColorPickerColor, input: any) {
   }
 
   if (typeof input === 'object') {
-    if (has(input, ['r', 'g', 'b'])) return stripAlpha(color.rgba, !input.a)
-    else if (has(input, ['h', 's', 'l'])) return stripAlpha(color.hsla, !input.a)
-    else if (has(input, ['h', 's', 'v'])) return stripAlpha(color.hsva, !input.a)
+    const shouldStrip = typeof input.a === 'number' && input.a === 0 ? !!input.a : !input.a
```

Comment:
Why is `typeof input.a === 'number'` necessary here, strict comparison to number checks type as well.